integration_tests/integration/notification-badge.spec.ts

```diff
@@ -44,14 +44,14 @@ context('Unallocated', () => {
   })
 
   it('must not show notification badge when no user preference is selected', () => {
-    cy.task('stubUserPreferenceTeams', [])
+    cy.task('stubUserPreferenceEmptyAllocationDemand')
```

Comment:
I might be missing something but don't we need a test for no user preference?

Reply:
the test is a bit ambiguously named, this is the response if there is no user preference ever or a cleared user preference